CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - `Recommender`s now share their core logic via their base class
+- Remove progressbar in examples
```

Comment:
@Scienfitz this currently fails due to the audit check:
![image](https://github.com/emdgroup/baybe/assets/84016733/179598fa-c69d-40b7-823e-a3941d7be341)
Is there anything we can do about this?

Reply:
you can/should excude the specified version from our pyptoject.toml file OR even better require a newer version (doesnt work if theres no newer version yet)

Reply:
try a la 
```
onnx = [
    "onnxruntime>=1.15.1",
    "skl2onnx>=1.15.0",
    "onnx!=0.15.0"
]
```

Reply:
What's really weird is that this version is already out since October 2023. Why does this only fail now? Doesn't this logically mean that the problem is probably due to another package?

Reply:
It seems like they plan to release a new version in March, let's discuss later what to do until then.
<img width="319" alt="image" src="https://github.com/emdgroup/baybe/assets/84016733/4aee9c97-5240-4502-af8f-e0477251d0e9">

Reply:
bug might have only been discovered now

Reply:
I'll create a Hotfix PR to ignore these vulnerabilities for now

Reply:
then no change should be done to .toml and the exceptions be put into tox (resolve if done)